.github/workflows/docker-release.yml

```diff
@@ -31,10 +30,22 @@ jobs:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ${{ env.IMAGE_NAME }}
+          tags: |
+            type=ref,event=tag,enable=${{ startsWith(github.ref, 'refs/tags/') }}
```

Comment:
The inconsistency in the `Git tag` with the docker hub pushed image tag, means reverting to the above. e.g `v1.3.2a` github tag outputs `mcsystems/news-search-api:1.3.2a0`, this is an issue since the `deploy.sh` expects the imaged to be tagged with the actual github tag

Reply:
Looking at the commit history, 👇🏽 is what you had set:
```yml
type=pep440,pattern={{version}}
```
i.e. you asked for version only and the action return `version` (without the `v`).

If you want the full tag as is i.e. you don't cate whether it's a valid version scheme or now, you can use `raw`:
```yml
type=pep440,pattern={{raw}}
```
If you wanted to make sure there is a valid version but still want the `v`, you can actually include it in the `pattern`:
```yml
type=pep440,pattern=v{{version}}
```

The linked doc has examples for all the above.

Reply:
@kilemensi  resolved with `type=pep440,pattern={{raw}}`